semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +120,48 @@ p strong {
   margin-left: 10px;
 }
 
+/*--------------------------------
+   Application home & create
+---------------------------------*/
+#application-area {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.company-title {
+  letter-spacing: @letterSpacingRegular;
+  line-height: @lineHeightBig;
```

Comment:
Same comment here about line height. If you're trying to make more space below, then `margin-bottom` is probably better. You only need to consider line height when considering multiple lines of the same text (and this company title should never span multiple lines).

Reply:
Cool, good point. I was just trying to bring what was in the invision to it, but I agree that for single lines that's probably not done using lineHeights, I'll remove those settings now.